packages/calypso-color-schemes/src/shared/color-schemes/_jetpack-cloud.scss

```diff
@@ -0,0 +1,124 @@
+.theme-jetpack-cloud,
```

Comment:
Currenly its both. I can remove the colourscheme part for now since there is no way to access it. 

What would be the more preferred way to include the colour scheme for jetpack cloud.